app/src/main/java/com/example/myapplication/NavGraph.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.myapplication
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Movie
+import androidx.compose.material.icons.filled.Person
+import com.example.myapplication.screens.*
+
+sealed class Screen(val route: String, val title: String, val icon: ImageVector? = null) {
+    object Movies : Screen("movies", "Movies", Icons.Default.Movie)
+    object Profile : Screen("profile", "Profile", Icons.Default.Person)
+    object Login : Screen("login", "Giriş Yap")
+    object MovieDetail : Screen("movie_detail/{title}/{year}/{rating}/{posterUrl}/{description}", "Movie Detail")
+}
```

Comment:
у класса Movie лучше добавить еще одно свойство/поле - Id.

если его нет в документации к api - то генерировать самим, если есть - получать через Api. 

тогда можно заменить такой длинный путь на:
```suggestion
    object MovieDetail : Screen("movie_detail/{movieId}, "Movie Detail")
}
```